test/test_quantization.py

```diff
@@ -775,6 +775,45 @@ def test_minmax_observer(self, qdtype, qscheme, reduce_range):
         self.assertEqual(qparams[1].item(), ref_zero_point)
         self.assertAlmostEqual(qparams[0].item(), ref_scale, delta=1e-5)
 
+    @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
+           qscheme=st.sampled_from((torch.per_channel_affine, torch.per_channel_symmetric)))
+    def test_per_channel_minmax_observer(self, qdtype, qscheme):
+        myobs = PerChannelMinMaxObserver(ch_axis=0, dtype=qdtype, qscheme=qscheme)
+        x = torch.tensor([[[[1.0, 2.0], [2.0, 2.5]], [[3.0, 4.0], [4.5, 6.0]]], [[[-4.0, -3.0], [5.0, 5.0]], [[6.0, 3.0], [7.0, 8.0]]]])
+        result = myobs(x)
+        self.assertEqual(result, x)
+        print("min_vals = ", myobs.min_vals)
+        print("max_vals = ", myobs.max_vals)
+        self.assertEqual(myobs.min_vals, [1.0, -4.0])
+        self.assertEqual(myobs.max_vals, [6.0, 8.0])
+        qparams = myobs.calculate_qparams()
+        if qscheme == torch.per_channel_symmetric:
+            ref_scales = [0.047058823529412, 0.062745098039216]
+            ref_zero_points = [0, 0] if qdtype is torch.qint8 else [128, 128]
+        else:
+            ref_scales = [0.023529411764706, 0.047058823529412]
+            ref_zero_points = [-128,  -43] if qdtype is torch.qint8 else [0, 85]
+        self.assertTrue(torch.allclose(qparams[0], torch.tensor(ref_scales, dtype=qparams[0].dtype)))
+        self.assertTrue(torch.allclose(qparams[1], torch.tensor(ref_zero_points, dtype=qparams[1].dtype)))
+
+    @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
+           qscheme=st.sampled_from((torch.per_channel_affine, torch.per_channel_symmetric)))
+    def test_per_channel_minmax_observer_scriptable(self, qdtype, qscheme):
+        obs = PerChannelMinMaxObserver(ch_axis=1, dtype=qdtype, qscheme=qscheme)
+        scripted = torch.jit.script(obs)
+
+        x = torch.rand(3, 4)
+        obs(x)
+        scripted(x)
+
+        self.assertEqual(obs.calculate_qparams(), scripted.calculate_qparams())
+
+        buf = io.BytesIO()
+        torch.jit.save(scripted, buf)
+        buf.seek(0)
+        loaded = torch.jit.load(buf)
+        self.assertEqual(obs.calculate_qparams(), loaded.calculate_qparams())
+
```

Comment:
This is a general request: We need to test that all observers are serializable. We currently only test for scriptability. This is needed for quantization aware training support. Can you add support in a seperate PR?
https://github.com/pytorch/pytorch/blob/9f6b6b81010e6f2edbf5faeb521284b1276ee1de/test/test_quantized_nn_mods.py#L254 to https://github.com/pytorch/pytorch/blob/9f6b6b81010e6f2edbf5faeb521284b1276ee1de/test/test_quantized_nn_mods.py#L303 provides a refefrence.